install/kubernetes/README.md

```diff
@@ -1,6 +1,6 @@
 # Install Istio on an existing Kubernetes cluster
 
-Please follow the installation instructions from [istio.io](https://istio.io/docs/tasks/installing-istio.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/).
```

Comment:
why not link directly into https://istio.io/docs/setup/kubernetes/quick-start.html ?